EnglishFilter/sections/antiadblock.txt

```diff
@@ -500,8 +500,8 @@ sozcu.com.tr,heraldo.es,20minutos.es,tucson.com,gendai.ismedia.jp,hamakei.com,te
 ||cdn.jsdelivr.net/gh/dewaplokis/block@master/dewablock.js
 ||cdn.jsdelivr.net/gh/*/js/arlinablock.js
 iptvbin.com$$script[tag-content="a.id=""arlinablock"""][max-length="1200"]
-||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$script,redirect=googlesyndication-adsbygoogle,important,domain=bemiraculous.tk|drivedrama.com|puressh.net|iptvbin.com
-@@||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$domain=bemiraculous.tk|drivedrama.com|puressh.net|iptvbin.com
+||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$script,redirect=googlesyndication-adsbygoogle,important,domain=htnovo.net,bemiraculous.tk|drivedrama.com|puressh.net|iptvbin.com
+@@||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$domain=htnovo.net,bemiraculous.tk|drivedrama.com|puressh.net|iptvbin.com
```

Comment:
please fix the typo, `|` should be user instead of `,`

Reply:
and please add new domains to `$domain` modifier to the end of the list

Reply:
Hehe Sorry,